geode-redis/src/acceptanceTest/java/session/NativeRedisSessionExpirationAcceptanceTest.java

```diff
@@ -50,4 +50,10 @@ public static void setup() {
   public void sessionShouldTimeout_whenAppFailsOverToAnotherRedisServer() {
     // Only using one server for Native Redis
   }
+
+  @Test
+  @Ignore
```

Comment:
Normally when I see an ignore on a test I want to know why the test is being ignored. I think it is something we could fix and then have this test. In this case I think we are overriding a test that we will never do with native redis. It seems like the empty method impl takes care of that and you would not need the ignore. This is just my opinion, I'd be okay if you leave the ignore if you think that is best.

Reply:
Just updated it and took out the ignores!